assets/js/base/context/providers/store-notices/components/store-notices-container.js

```diff
@@ -74,11 +66,7 @@ export const StoreNoticesContainer = ( {
 						}
 					} }
 				>
-					<span
-						dangerouslySetInnerHTML={ sanitizeHTML(
-							props.content
-						) }
-					/>
+					{ sanitizeHTML( props.content ) }
```

Comment:
Shouldn't `{ sanitizeHTML( props.content ) }` remain `<span dangerouslySetInnerHTML={ { __html: sanitizeHTML( props.content ) } } />` to ensure that HTML elements are visible as expected? Wouldn't changing that part to `{ sanitizeHTML( props.content ) }` show the plain HTML code on the site?

I'm asking, because when I replace
```js
{ ( showItems || collapsible ) && (
  <div
    className="wc-block-components-shipping-rates-control__package-title"
    dangerouslySetInnerHTML={ {
      __html: sanitizeHTML( packageData.name ),
    } }
  />
) }
```
with
```
{
  ( showItems || collapsible ) && sanitizeHTML( packageData.name )
}
```
in https://github.com/woocommerce/woocommerce-blocks/blob/2f8e4f79b01ef706723164cc1e72ddabad04f6d6/assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx#L73-L80, I see the following result:

<img width="401" alt="Screenshot 2022-09-28 at 11 43 00" src="https://user-images.githubusercontent.com/3323310/192689310-2a22a748-3359-4d78-bf11-b2889699f045.png">

Reply:
@nielslange  That's a good question.  So we use `sanitizeHTML( props.content )` here because `Notice` has changed the behavior and magically deals with the `dangerouslySetInnerHTML` for us.  If we keep using the below code, we will actually get an empty div tag which causes the store notices component doesn't render the text.
```
<span
	dangerouslySetInnerHTML={ sanitizeHTML( props.content ) }
/>
```

Reply:
For your case, you are not using `Notice`, so you still need the `dangerouslySetInnerHTML` and pass an object with a `__html` key to translate your sanitized HTML.

Reply:
Thanks for clarifying this, @hsingyuc.